app/controllers/admin/categories_controller.rb

```diff
@@ -0,0 +1,68 @@
+module Admin
+  class CategoriesController < Admin::BaseController
+    before_action :set_category, only: %i[show edit destroy update]
+
+    def index
+      @pagy, @category = pagy(Category.all, items: 5)
+      respond_to do |format|
+        format.html
+        format.csv { send_data Category.all.to_csv, filename: "categories-#{Date.today}.csv" }
+      end
+    end
+
+    def show; end
+
+    def new
+      @category = Category.new
+    end
+
+    def create
+      @category = Category.new(category_params)
+      respond_to do |format|
```

Comment:
If we are expecting only one HTML format, in that case, no need for `respond_to` block